GenumKit/Stencil/Contexts.swift

```diff
@@ -131,3 +131,30 @@ extension StringsFileParser {
     return Context(dictionary: ["enumName": enumName, "tableName": tableName, "strings": strings])
   }
 }
+
+/* MARK: - Stencil Context for Fonts
+    - `enumName`: `String`
+    - `fonts`: `Array`
+        - `familyName`: `String`
+        - `variant`: `Array`
+            - `name`: `String`
+            - `value`: `String`
+*/
+
+extension FontsFileParser {
+    public func stencilContext(enumName enumName: String = "Family") -> Context{
+        // turn into array of dictionaries
+        let fontFamilies = entries.map { (name: String, variants: Set<String>) -> [String:AnyObject] in
+            let fontVariants = variants.map { return [
+                "name" : $0,
+                "value" : ($0 == "Regular" ? name : "\(name)-\($0)")
```

Comment:
Be careful, to construct a `UIFont` you must pass the font **postscript name** (which is not always `Family-Variant`!)
For example for `/Library/Fonts/Hannotate.ttc`  the family name is `"Hannotate SC"`, the style/variant name is `"Regular"` but the PostScript name is `"HannotateSC-W5"`! (yeah, fonts are hard)

I suggest you add a `postscriptName` entry to your `Font` tuple, fill it with the value or `CTFontCopyPostScriptName(font) as String` when parsing the font file in the function `CTFont.parseFontInfo` I gave in the comment above, and use that to fill the StencilContext instead.

```
fonts: Array
 - family: String   // Like "Arial"
 - variants: Array
    - style: String  // Like "Bold Italic"
    - name: String // Postscript name to use to build UIFont, like "Arial-BoldItalicMT"
```